master/buildbot/steps/source/git.py

```diff
@@ -28,11 +28,12 @@
 from buildbot.process import remotecommand
 from buildbot.process.properties import Properties
 from buildbot.steps.source.base import Source
-from buildbot.util.git import GitMixin
+from buildbot.steps.worker import CompositeStepMixin
+from buildbot.util.git import GitWorkerCommonMixin
```

Comment:
Common and Mixin looks like redounded. I would say GitStepMixin

Reply:
Fixed now.